modules/gapi/include/opencv2/gapi/render.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef OPENCV_GAPI_RENDER_HPP
+#define OPENCV_GAPI_RENDER_HPP
+
+#include <string>
+#include <vector>
+
+#include <opencv2/gapi/opencv_includes.hpp>
+#include <opencv2/gapi/util/variant.hpp>
+#include <opencv2/gapi/own/exports.hpp>
+
+namespace cv
+{
+namespace gapi
+{
+
+struct TextEvent
+{
+    std::string text;
+    float x;
+    float y;
```

Comment:
Maybe cv::Point2f is better?

Reply:
I think it is better not to make an explicit dependence on OpenCV.
In this case it is better to delete cv::Scalar.

Reply:
This part can depend on OpenCV anyway. There's no plan to expose it in Standalone mode.

Reply:
Fixed